packages/core/src/env-set/oidc.ts

```diff
@@ -87,18 +87,20 @@ export const readPrivateKeys = async (): Promise<string[]> => {
   const notExistPrivateKeys = privateKeyPaths.filter((path): boolean => !existsSync(path));
 
   if (notExistPrivateKeys.length > 0) {
-    const notExistPrivateKeysRawValue = JSON.stringify(notExistPrivateKeys);
     throw new Error(
-      `Private keys ${notExistPrivateKeysRawValue} configured in env \`OIDC_PRIVATE_KEY_PATHS\` not found.`
+      `Private keys ${notExistPrivateKeys.join(
+        ' and '
+      )} configured in env \`OIDC_PRIVATE_KEY_PATHS\` not found.`
```

Comment:
If we have more than 2 `nonExistPrivateKeys`, then the result would be `key 1 and key2 and key3`, which is not good IMO.
Better check with the `Intl.ListFormat` API? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat